src/driver/src/DefaultDriver.cpp

```diff
@@ -188,58 +197,82 @@ std::string normalizeOutputFile(const std::string& str) {
 }  // namespace
 
 
-genny::driver::DefaultDriver::ProgramOptions::ProgramOptions(int argc, char** argv) {
+DefaultDriver::ProgramOptions::ProgramOptions(int argc, char** argv) {
     namespace po = boost::program_options;
 
-    po::options_description description{u8"🧞‍ Allowed Options 🧞‍"};
+    po::options_description progDescription{u8"🧞‍ Allowed Options 🧞‍"};
     po::positional_options_description positional;
 
     // clang-format off
-    description.add_options()
-        ("help,h",
-            "Show help message")
-        ("list-actors",
-            "List all actors available for use")
-        ("dry-run",
-            "Exit before the run step---"
-            "this may still make network connections during workload initialization")
-        ("metrics-format,m",
+    const auto runModeHelp = u8R"(
+Genny run modes
+    normal
+        Run the workload normally; default mode if no run mode is specified
+
+    dry-run
+        Exit before the run step -- this may still make network connections during workload initialization
+
+    evaluate
+        Print the evaluated YAML workload file with minimal validation
+
+    list-actors
+        List all actors available for use
+    )";
+
+    progDescription.add_options()
+            ("help,h",
+             "Show help message")
+            ("run-mode", po::value<std::string>()->default_value("normal"),
+             runModeHelp)
+            ("list-actors",
+                    "DEPRECATED. Please use the \"list-actors\" run mode.")
+            ("dry-run",
+                    "DEPRECATED. Please the \"dry-run\" run mode.")
```

Comment:
imho it's fine to break old usages at least for now - I'd just remove these for now - the change to DSI is relatively easy to do simultaneously when merging this.